packages/sdk-cli/package.json

```diff
@@ -22,6 +22,7 @@
     "@fitbit/portable-pixmap": "^1.0.3",
     "@moleculer/vorpal": "^1.11.5",
     "@openid/appauth": "^1.2.8",
+    "async-wait-until": "^2.0.8",
```

Comment:
Looks like you've replaced `p-wait-for` but left all the Jest changes relevant to it in place, you should revert those if they're no longer necessary, particularly since they make reference to the unused/uninstalled package.